book/src/api-lighthouse.md

```diff
@@ -177,3 +177,13 @@ See [Validator Inclusion APIs](./validator-inclusion.md).
 ### `/lighthouse/validator_inclusion/{epoch}/global`
 
 See [Validator Inclusion APIs](./validator-inclusion.md).
+
+### `/lighthouse/beacon/states/{state_id}/ssz`
```

Comment:
Adding the state_id options here might be good, or referencing the eth2 api. Might not be clear otherwise.